modules/ROOT/pages/connected-apps-devs-end-users.adoc

```diff
@@ -0,0 +1,20 @@
+= Connected Apps for Developers and End Users
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Developers
```

Comment:
Please link and explain https://anypoint.mulesoft.com/accounts/api/v2/oauth2/.well-known/openid-configuration -- Developers needs this as well as the URLs that it contains to function properly.

An e2e example of how to set up a connected app as a Developer would be useful here too.

Please also list what parts of the spec and protocols we're compliant with